### DISCUSSION TITLE
GitHub Lists not visible enough

### DISCUSSION THREAD

**Discussion by loftwah** [community/community#20602] [Other Feature Feedback, Questions, & Ideas]
I have been using GitHub Lists for a little while, but I lost track of where to actually find them for a while and had to really go digging to find it (easy with a quick search) but I think it could be more visible, like giving it its own menu item or something. It does fit well in the stars section, but I forgot it would be located there.

**Comment by thenithinbalaji** [comment]
I think profile badges and lists are getting a separate tab. Both are in beta in btw.
I think this because,
Lists can be accessed from this URL
```
https://github.com/stars/<yourusername>/lists/<listname>
```

Badges can be accessed from this URL
```
https://github.com/<yourusername>?tab=achievements
```

So, it's highly likely for them to get separate tabs.
